backend/views/error.jade

```diff
@@ -0,0 +1,6 @@
+extends layout
+
+block content
+  h1= message
+  h2= error.status
+  pre #{error.stack}
```

Comment:
Out of curiosity, can you explain what the `.jade` files are?

Reply:
not too sure, these were automatically generated by express generator. Looks like they're templates files that create HTML files

Reply:
Ah makes sense, thanks!